sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -98,7 +99,23 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     programOffering.creator = { id: userId } as User;
     // When creating a new offering, parent id and the primary id are the same.
     programOffering.parentOffering = programOffering;
-    return this.repo.save(programOffering);
+
+    try {
+      return await this.repo.save(programOffering);
```

Comment:
`await` is not needed here.

Reply:
If I remove the await the promise will be returned and the thrown error is not caught. Is there any other way to solve this?

Reply:
My bad!